package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "sse-stream",
+  "name": "@davidmarkclements/sse-stream",
```

Comment:
I think that's an accident - must have published my own whilst waiting for a merge and accidentally committed package.json alterations

advise that it's merged and fixed